app/client/src/components/shared/AlertMessage/index.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { globalHistory } from '@reach/router';
+// styles
+import { colors } from 'styles/index.js';
+// contexts
+import { useNotificationsContext } from 'contexts/LookupFiles';
+
+// --- components ---
+type Props = {};
+
+// --- styled components ---
+const Banner = styled.div`
+  background-color: ${(props) => props.backgroundColor};
+  color: ${(props) => props.color};
+  width: 100%;
+  margin: 0 auto;
+  padding: 10px 5px;
+  text-align: center;
+
+  p {
+    padding: 0;
+    margin: 0;
+  }
+`;
+
+const Separator = styled.hr`
+  margin: 0;
+  background-color: ${colors.gray9};
+`;
+
+function AlertMessage({ ...props }: Props) {
+  const notifications = useNotificationsContext();
+
+  const [pathname, setPathname] = React.useState('');
+
+  if (notifications.status === 'failure') return null;
+
+  if (notifications.status === 'fetching') return null;
+
+  if (
+    notifications.status === 'success' &&
+    (!notifications.data || Object.keys(notifications.data).length === 0)
+  ) {
+    return null;
+  }
+
+  if (
+    notifications.status === 'success' &&
+    Object.keys(notifications.data).length > 0
+  ) {
+    const data = notifications.data;
```

Comment:
This suggestion just cleans up the if statements a little bit. 

```suggestion
 
  if (notifications.status === 'success') {
    if (!notifications.data || Object.keys(notifications.data).length === 0) {
      return null;
    }

    const data = notifications.data;
```

Reply:
thanks, I've simplified the if statements as you recommended